tests/test_cow_storage_pool.py

```
@@ -0,0 +1,260 @@
+        esac
+        return 0
+    }
+""")
+
+
+class TestInstallScriptIntegrity:
+    """Validate the install script is well-formed."""
+
+    def test_syntax_valid(self):
+        """The install script must pass bash -n."""
+        rc, out, err = _bash_run(f"bash -n {INSTALL_SCRIPT}")
```

Ruff (0.15.14)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)